src/main/resources/org/jenkinsci/plugins/durabletask/Messages.properties

```diff
@@ -1,2 +1,3 @@
 BourneShellScript.bourne_shell=Bourne shell
 WindowsBatchScript.windows_batch=Windows batch
+PowershellScript.powershell=Powershell script
```

Comment:
**PowerShell** acc. to https://msdn.microsoft.com/powershell

(though this UI is not generally shown anyway)